src/components/main-view.tsx

```diff
@@ -6,38 +6,53 @@ import { ForceArrows } from "./force-arrows";
 import "./main-view.scss";
 
 interface IProps {
+  showForce: boolean;
+  version: string;
   saveToTable: (object1: SysObject, object2: SysObject) => void;
   clearTable: () => void;
+  handleChangeShowForceState: (state: boolean) => void;
 }
 
-export const MainView: React.FC<IProps> = ({ saveToTable, clearTable }) => {
+export const MainView: React.FC<IProps> = ({ showForce, version, saveToTable, clearTable, handleChangeShowForceState }) => {
   const [ object1, setObject1 ] = useState<SysObject>(SysObject.bottle);
   const [ object2, setObject2 ] = useState<SysObject>(SysObject.earth);
 
   const handleSave = () => {
     saveToTable(object1, object2);
+    handleChangeShowForceState(true);
+  };
+
+  const showAlert = () => {
+    alert("Click on Calculate forces button to see a representation of the forces between the two objects");
   };
 
   return (
     <div className="main-view">
       <div className="header">Select two objects:</div>
       <div className="columns">
         <div className="column">
-          <div>Object 1: <SelectObject value={object1} onChange={setObject1}/></div>
+          <div data-testid="object-1">Object 1: <SelectObject value={object1} onChange={setObject1} changeShowForceState={handleChangeShowForceState}/></div>
           <ObjectSymbol objectType={object1} />
         </div>
         <div className="column arrows">
           <div className="label">strength and direction<br/>of gravitational forces</div>
-          <ForceArrows object1={object1} object2={object2} />
+          { version === "forces-unknown" && !showForce
```

Comment:
`version` could be typed to some explicit, allowed values and `getVersion` could check the params if they match allowed values. But since we deal just with one version, I don't think it matters. Just mentioning it if there's a plan to add more versions and more params.